hbase_sql/src/main/java/com/splicemachine/olap/OlapServerZNode.java

```diff
@@ -0,0 +1,54 @@
+package com.splicemachine.olap;
+
+import java.util.Objects;
+
+public class OlapServerZNode implements Comparable<OlapServerZNode> {
+    private String queueName;
+    private int sequence;
+
+    public OlapServerZNode(String queueName, int sequence) {
+        this.queueName = queueName;
+        this.sequence = sequence;
+    }
+
+    public String getQueueName() {
+        return queueName;
+    }
+
+    public long getSequence() {
+        return sequence;
+    }
+
+    public String toZNode() {
```

Comment:
looks good, although i don't understand everything. are server names always of the form xxxxNUM, where NUM is 10 digits?

Reply:
To avoid duplicates each OlapServer registers itself in ZooKeeper using a sequence node, which appends a unique identifier at the end to guarantee uniqueness.